R/merge.R

```diff
@@ -103,6 +103,16 @@ setGeneric("mergeSamples",
            function(x, f, archetype = 1L, ...)
                standardGeneric("mergeSamples"))
 
+# Helper function to check if 'f' is a valid column name in rowData or colData
+.is_valid_column_name <- function(x, f, dim_type) {
+    if (dim_type == "rows") {
```

Comment:
by the way could we easilyl use `dim.type` instead of `dim_type`?

Reply:
Why we don't here use MARGIN (as usual for rows vs. columns; or features and samples)? See other functions for such cases. Consistency could be desirable.

Reply:
Oh this function is removed. I was testing several approaches.